### PR TITLE
Logout button added

### PR DESCRIPTION
**Pull request by dinurymomshad**
When I logged in I was having a hard time to logout. So to ease development, I added a Appbar in Calendar screen and added a logout button there.

Also refactored calendar_page and gradient_button file.

### CODE REVIEW

File: app/exerlog/lib/UI/calendar/calendar_page.dart [hunk 1]
```diff
@@ -16,40 +19,81 @@ class _CalendarPageState extends State<CalendarPage> {
...
             )
           ],
-          ),
+        ),
       ),
     );
   }
+}
+
+class _Logout extends StatefulWidget {
```
Comment (Scount) [inline app/exerlog/lib/UI/calendar/calendar_page.dart:58]: Please refactor this widget into a seperate file like 'logout_button.dart'.
Also change the widget name into the same name as the file like 'LogoutButton'.

File: app/exerlog/lib/UI/calendar/calendar_page.dart [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import 'package:exerlog/UI/global.dart';
 import 'package:exerlog/UI/gradient_button.dart';
 import 'package:exerlog/UI/workout/workout_page.dart';
+import 'package:exerlog/main.dart';
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
+
 import 'calendar_widget.dart';
 
 class CalendarPage extends StatefulWidget {
```
Comment (Scount) [inline app/exerlog/lib/UI/calendar/calendar_page.dart:10]: Please change this widget from a StatefulWidget into a StatelessWidget. This widget isn't using a setState and therefore won't change its state.
```suggestion
class CalendarPage extends StatelessWidget {

```

**Review by Scount — COMMENTED** [review]
I like the idea of having a logout button.
One general point, maybe remove all unused imports in files you touched. It will help to have a clean style.

**Comment by dinurymomshad** [comment]
@KalleHallden Can you check this out? I resolved the merge conflicts

**Pull request merged**